README-CN.md

```diff
@@ -1,14 +1,16 @@
 # Apache RocketMQ 客户端
 
 [![License][license-image]][license-url]
-[![CPP][cpp-image]][cpp-url]
```

Comment:
GitHub badge takes up too much space, it is suggested to put it in one line or in the subdirectory.